### DISCUSSION TITLE
Birdnet docker into unraid

### DISCUSSION THREAD

**Discussion by killervette42** [alexbelgium/hassio-addons#1393] [General]
I would love to get the Birdnet-pi docker into unraid. Can anyone help me?

**Comment by alexbelgium** [comment]
Hi, a specific build is likely needed using the same dockerfile, but removing the HA specificities.  We could even imagine reusing the same dockerfile, but it would be the build script that either adapts it to HA by using the config.yaml to build, or adapts it to unraid.

HA specificities to adapt would be everything within the config.yaml : folders, ports and variables mapping probably
